magento/magento2
# Page heading null issue fixed

### Description (*)

Issue Fix - CMS Page throwing error if no page title present 



### Fixed Issues (if relevant)

1. Fixes #39092

### Manual testing scenarios (*)

1. Create New CMS Page without Title
2. Verify if the page is loading or not.



### Contribution checklist (*)
